src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -46,15 +46,20 @@ function writeDrupalDebugPage(files) {
 
 // Creates the file object to add to the file list using the page and layout
 function createFileObj(page, layout) {
+  // Exclude some types from sitemap.
+  const privateTypes = ['outreach_asset', 'person_profile', 'support_service'];
```

Comment:
You don't have to change this, but the new JS Set class is good for this kind of check:

```js
const privateTypes = new Set(['outreach_asset', 'person_profile', 'support_service']);
const privStatus = privateTypes.has(page.entityBundle);
```